Plugins/Public/base_plugin/ClientCommands.cpp

```diff
@@ -51,7 +51,7 @@ void SendMarketGoodUpdated(PlayerBase *base, uint good, MARKET_ITEM &item)
 				else
 				{
 					_snwprintf(buf, sizeof(buf), L" SetMarketOverride %u %u %f %u %u",
-						base->proxy_base, good, item.price, 0, item.quantity);
```

Comment:
How about renaming item.price to more appropriate item.buyprice then?

Reply:
yes that makes sense. We can do this